pkg/eviction/evict.go

```diff
@@ -2,19 +2,19 @@ package eviction
 
 import (
 	"context"
-	"sync"
+	"fmt"
 
+	"golang.org/x/sync/singleflight"
 	policyv1 "k8s.io/api/policy/v1"
 	policyv1beta1 "k8s.io/api/policy/v1beta1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 )
 
 var (
-	canEvictV1      = true
-	canEvictV1beta1 = true
-	lock            = sync.Mutex{}
+	evictVersion = ""
+	isInit       = false
```

Comment:
sorry, last bike shedding: I was a bit confused about the name`isInit`. Please something like `initDone` or `isInitialized` or `isInited`, I don't care, but `isInit` to me is always "is currently initializing".